core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -105,14 +83,14 @@
    */
   public static final String FIELD_COUNT_PROPERTY_DEFAULT = "10";
 
-  int fieldcount;
+  private int fieldcount;
```

Comment:
I found a way to allow protected variables in checkstyle.xml. It allowed me to set the variables to be protected.